contents/docs/deployment/deploy-heroku.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Deploying to Heroku
+sidebar: Docs
+showTitle: true
+---
+
+## Why Heroku
+
+Heroku is currently the **quickest** way to get a production PostHog environment up and running!
+
+If you've never heard of Heroku or what it does, feel free to check out [this doc](https://www.heroku.com/about) that provides a quick gist of the product.
+
+## Prerequisites
+
+- A [Heroku](https://signup.heroku.com/) account set up and verified *(it's free, don't worry :))*.
+    > _Although the deployment is free, ensure that your **credit card information** is entered in your profile. If not, the initial build will fail and prompt you to do so._
+
+## Step By Step Installation
+
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/posthog/posthog)
+
+1. Click on the button above to go to the app creation screen.
+
+2. Enter the desired name of your PostHog app and hit **Deploy App** on the bottom of the screen.
+
+3. Wait.
+
+    > _Note: This may take a few minutes, so resist the urge to refresh the page or restart the process_
+
+4. Once completed, you will see two options on the bottom of the screen: **Manage App** & **View App**. To go to PostHog, simply click the **View App** button!
+
+    - If you want to review add ons and other details of the instance, simply click the **Manage App** button or access the instance from your **Dashboard** *(click on the 9 dots by your profile on the top right)*.
+
+    - Within the **Manage App** screen, simply click **Open App** on the top right to start your PostHog environment.
+
+## Add-Ons
+
+By default, we install a `hobby-dev` Postgres and Redis add-on to the app - these can be found in the **Manage App** screen under **Installed add-ons**.
+
+### Upgrading From Free Tier
+
+Since `hobby-dev` is the free tier, there may be significant lag if you visit the app after your site has not been visited for a while.
+
+To avoid this, you can upgrade up from the free tier; this can be done through the same **Manage App** screen by clicking on the *Configure Add-ons* link by the **Installed add-ons** section. Once in the add-ons screen:
+
+- To upgrade your Redis plan, **click on the button** to the right of both the `Heroku Postgres` and `Heroku Redis` add-on (it looks like an up and down symbol you'd see by an elevator). From here, click on `Modify Plan` and pick your desired one!
```

Comment:
The free Redis plan tends to be fine for most people. It's more the dyno that needs to be updated

Reply:
Should I remove the redis plan bullet then or keep it in? Also, do you want me to add a specific section about updating dynos?

Reply:
Maybe add (Optional) to redis. This does need a specific section on how to upgrade dynos unless I'm missing something?

Reply:
I added a dyno section but i left the redis bullet. I tried putting an optional tag on it but it made it seem as if upgrading postgres was necessary which was misguiding.

Reply:
ah wait maybe we're talking about different things, but the way I'm reading this sounds like Redis and Postgres are required upgrades and a dyno is optional, but in reality it's the other way around. You definitely _need_ to upgrade a dyno, Postgres and redis only need to be upgraded when you reach their limits